src/tests/Microsoft.PowerFx.Interpreter.Tests/RecalcEngineTests.cs

```diff
@@ -512,6 +512,18 @@ public void EmptyEnumStoreTest()
 
             Assert.True(enums.Count() > 0);
         }
+        
+        /// <summary>
+        /// Issue #263 - Filter should be able to handle empty rows
+        /// </summary>
+        [Fact]
+        public void BlankRowFilter()
+        {
+            var engine = new RecalcEngine();
+            var result = engine.Eval("Index(Filter(Table({a:1},{a:2},If(1<0,{a:3}),{a:4},{a:5}), IsBlank(a) Or a > 0),3)");

```

Comment:
>Filter(Table({a:1},{a:2},If(1<0,{a:3}),{a:4},{a:5}), IsBlank(a) Or a > 0),3) [](http://example.com/codeflow?start=44&length=76)

Can you add this as a test case here instead? 
The .txt cases are easier to manage

[https://github.com/microsoft/Power-Fx/blob/main/src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/FilterFunctions.txt](https://github.com/microsoft/Power-Fx/blob/main/src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/FilterFunctions.txt)

That will also let you avoid Index() and verify the whole table (see other tests in that file)

Those .txt are run through a special runner here: [https://github.com/microsoft/Power-Fx/blob/4e57e10da30e19025b9c35885bfb99571404639f/src/tests/Microsoft.PowerFx.Interpreter.Tests/FileExpressionEvaluationTests.cs#L24](https://github.com/microsoft/Power-Fx/blob/4e57e10da30e19025b9c35885bfb99571404639f/src/tests/Microsoft.PowerFx.Interpreter.Tests/FileExpressionEvaluationTests.cs#L24)